logger/xp.py

```diff
@@ -1,7 +1,7 @@
 import copy
 import git
 import time
-import cPickle as pickle
+import _pickle as pickle
```

Comment:
Could you wrap that around something that allows python2 retro-compatibility? Like :
```
try:
    import _pickle as pickle
except:
    import cPickle as pickle
```